chart/templates/NOTES.txt

```diff
@@ -7,10 +7,12 @@ Basicauth username: {{ .Values.nginx.basicauth.credentials.username }}
 Basicauth password: {{ .Values.nginx.basicauth.credentials.password }}
 {{- end }}
 
+{{- if and (.Values.shell.enabled) (ne .Values.referenceData.referenceEnvironment .Values.environmentName) }}
```

Comment:
I would only check for `.Values.shell.enabled`, otherwise it's impossible to get a shell container for the reference environment. If the reference environment is the production environment, we would have a dedicated configuration file, and we could set `shell.enabled: false` there.

Reply:
Fixed